lib/options.nix

```
@@ -220,16 +326,28 @@ rec {
           (if isList example then "${pkgsText}." + concatStringsSep "." example else example);
       });
 
-  /* Alias of mkPackageOption. Previously used to create options with markdown
-     documentation, which is no longer required.
+  /**
+    Alias of mkPackageOption.
+
+    Previously used to create options with markdown
+    documentation, which is no longer required.
   */
   mkPackageOptionMD = mkPackageOption;
 
-  /* This option accepts anything, but it does not produce any result.
+  /**
+    This option accepts anything, but it does not produce any result.
+
+    This is useful for sharing a module across different module sets
+    without having to implement similar features as long as the
+    values of the options are not accessed.
 
-     This is useful for sharing a module across different module sets
-     without having to implement similar features as long as the
-     values of the options are not accessed. */
+
+    # Inputs
+
+    `attrs`
+
+    : 1\. Function argument
```

What is `1\.` for?
Why are we repeating `attrs`, which is just an implementation detail, and not actually referenced in the documentation?

```suggestion
    : Attribute set whose attributes override the argument to `mkOption`.
```

---

> What is 1\. for?

Escaped argument list. This is autogenerated if there is no documentation.
Since the initial goal was to migrate to commonmark my migration tool creates such lists, which need to be escaped in order to not conflict with definition list syntax.

It is essentially a placeholder until somebody documents it.